tests/integration.spec.ts

```diff
@@ -108,6 +108,28 @@ describe('jmap-client-ts', () => {
     expect(emailGetResponse.accountId).toBeDefined();
   });
 
+  it('should have email_changes working', async () => {
+    const getResponse = await client.email_get({
+      accountId: client.getAccountIds()[0],
+      ids: [],
```

Comment:
In JMAP semantic `[]` (get nothing) have a different meaning that `null` (get all the universe)

Reply:
Yes, and that is the aim of this request, to get the state of emails.